tests/robot-tests/run_tests.py

```diff
@@ -160,20 +165,26 @@
 else:
     load_dotenv(".env." + args.env)
 
-assert os.getenv("TIMEOUT") is not None
-assert os.getenv("IMPLICIT_WAIT") is not None
-assert os.getenv("PUBLIC_URL") is not None
-assert os.getenv("ADMIN_URL") is not None
-assert os.getenv("PUBLIC_AUTH_USER") is not None
-assert os.getenv("PUBLIC_AUTH_PASSWORD") is not None
-assert os.getenv("RELEASE_COMPLETE_WAIT") is not None
-assert os.getenv("WAIT_MEDIUM") is not None
-assert os.getenv("WAIT_LONG") is not None
-assert os.getenv("WAIT_SMALL") is not None
-assert os.getenv("FAIL_TEST_SUITES_FAST") is not None
-assert os.getenv("IDENTITY_PROVIDER") is not None
-assert os.getenv("WAIT_MEMORY_CACHE_EXPIRY") is not None
-assert os.getenv("EXPIRED_INVITE_USER_EMAIL") is not None
+
+required_env_vars = [
```

Comment:
Hey, just wondering if we need to add the following to this list too?

ADMIN_EMAIL
ADMIN_PASSWORD
EXPIRED_INVITE_USER_PASSWORD

Reply:
Gotcha, looks good to me then